draft-ion.adoc

```diff
@@ -52,50 +58,59 @@ However, JSON's simple grammar can support virtually any data model - additional
 context is required for machine clients and servers to interpret data and
 metadata in important ways.
 
-Ion exists to codify specific name/value pairs within a JSON object to better
-facilitate JSON-based data and metadata exchange between machine clients and
-servers. As such, Ion does not define a new data grammar - it is a
-Media Subtype of `application/json`.
+Ion codifies specific name/value pairs within JSON objects to better
+facilitate JSON-based data and metadata exchange between machines.  And while machine interchange is the primary goal, Ion's second most important design goal is to enable intuitive and context-relevant structure so Ion content is very easy to understand for humans, even if they do not know what Ion is.  As such, Ion does not define a new data grammar - it is a Media Subtype of `application/json`.
 
 === Terminology
 
 These terms are defined by this specification:
 
+.Member
+A JSON name/value pair as defined in https://tools.ietf.org/html/rfc7159#section-4[RFC 7159].
+
 .Metadata
-Data about a resource.
+A set of data that provides information about other data.
 
 .Meta Object
-A JSON object that contains resource metadata.
+A JSON object with one or more members that all represent Metadata.
+
+.Root Object
+The single JSON object at the root of an Ion content structure.
```

Comment:
Right. My point was massaging the wording a little to make that a little more explicitly clear on the motivation since at least one data point (me) took a while to figure out why for that definition (and I actually did not understand it until later in the spec when you referred to  the Root Object). It makes total sense, but is mildly subtle. Not a major issue. Just pointing it out if we want to address it.

So, maybe:

A single JSON object at the root of an Ion content structure that can contain metadata.

Or something like that. There are so many people that never think about this and slug around {[]} as resources. Total anti-pattern. Anyhow, like I said. Not a blocker but it is actually a really important concept in general for good API design IMO (and I think yours :wink: ).